streamly-coreutils.cabal

```diff
@@ -47,6 +47,18 @@ common compile-options
     if os(windows)
       cpp-options:    -DCABAL_OS_WINDOWS
 
+    ghc-options:      -Weverything
```

Comment:
-- We do not get warnings for missing top level
                        -- signatures when this is on.
                        -Wno-missing-exported-signatures